drivers/net/phy/micrel.c

```diff
@@ -577,6 +577,13 @@ static int ksz8041_fiber_mode(struct phy_device *phydev)
 	return of_property_read_bool(of_node, "micrel,fiber-mode");
 }
 
+static int ksz8081_is_enable_broadcast_address(struct phy_device *phydev)
+{
+	struct device_node *of_node = phydev->mdio.dev.of_node;
+
+	return of_property_read_bool(of_node, "micrel,ksz8081-enable-broadcast-address");
```

Comment:
Lets rename to "micrel,kszphy-allow-broadcast-address".  HW has the broadcast address enabled by default, but this driver disables it for all phys. We are adding a new option to allow the broadcast address to remain enabled.